SDLTooSharp/Managed/Font/IFont.cs

```diff
@@ -5,5 +5,5 @@ namespace SDLTooSharp.Managed.Font;
 
 public interface IFont
 {
-    public SDLSurface RenderGlyphLCD(uint glyph, Color foreground, Color background);
+    public ISurface RenderGlyphLCD(uint glyph, Color foreground, Color background);
```

Comment:
We could consider making the `RenderGlyphLCD` method asynchronous if it involves intensive computation or resource loading. This may improve responsiveness in applications that leverage this interface.